### DISCUSSION TITLE
Asynchronous actions

### DISCUSSION THREAD

**Discussion by robsonsobral** [robertodoering/harpy#671] [Ideas]
I'm getting some warnings of API limit reached and this gave me an idea.

The requests to like or retweet could be kept on a queue until next API update. The buttons could stay pulsing or faded until the related request be concluded.

Too weird?

**Comment by robertodoering** [comment]
Since likes are limited app wide by 1000 requests in 24 hrs, this won't work unfortunately.
Using a queue for all users would just delay likes by an ever growing amount. Imagine liking a tweet in harpy but it takes a day or more until it's actually liked - not a great workaround unfortunately

**Reply by robsonsobral** [reply]
"app wide by 1000"? Really? REALLY? Wow!

Thank you so much for explain this, @robertodoering .

(How do I close this thing?)

**Reply by robertodoering** [reply]
Yeah, unfortunately :(

You can mark my post as the answer but you don't need to close the discussion

**Reply by yoshimo** [reply]
Nobody would accept that for the official twitter app.
